packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -82,31 +102,53 @@ class DevToolsAppState extends State<DevToolsApp> {
   }
 
   /// The routes that the app exposes.
-  final Map<String, UrlParametersBuilder> _routes = {
-    '/': (_, params) => Initializer(
-          url: params['uri'],
-          builder: (_) => DevToolsScaffold(
-            tabs: const [
-              InspectorScreen(),
-              TimelineScreen(),
-              MemoryScreen(),
-              PerformanceScreen(),
-              DebuggerScreen(),
-              if (showNetworkPage) NetworkScreen(),
-              LoggingScreen(),
-              InfoScreen(),
-            ],
-            actions: [
-              HotReloadButton(),
-              HotRestartButton(),
-              OpenSettingsAction(),
-              OpenAboutAction(),
-            ],
-          ),
-        ),
-    '/connect': (_, __) =>
-        DevToolsScaffold.withChild(child: ConnectScreenBody()),
-  };
+  Map<String, UrlParametersBuilder> get routes => _routes ??= {
```

Comment:
nit: consider using `{` instead of `=>` in this case to make the code easier to read.